### ISSUE TITLE
MIssing detail from SmartSolar RS 450/100

### ISSUE THREAD

**Issue by morphias2004** [kwindrem/SetupHelper#14]
Just installed GuiMods and am seeing the following for the MPPT:

![image](https://user-images.githubusercontent.com/68760693/164152013-6474f8b6-44f9-489f-905e-a4dae2a603d7.png)

This is on the Cerbo GX I was just working on you with in the SetupHelper install issue thread.

The Cerbo is now on v2.85.

The MPPT is connected via VE.Can

![image](https://user-images.githubusercontent.com/68760693/164152272-f158ddcf-2f19-434e-a498-6b2f87aacfef.png)

I'm wondering if it's because the RS models have 2 internal MPPT's?

Let me know any additional details you need me to get.

**Comment by kwindrem** [comment]
Yes, the issue is the RS MPPT has two internal trackers that are reported separately. GuiMods shows PV voltage and current so in this case there are two separate value pairs. The tile supports space for 2-3 PV chargers but in this case I need to use more space to report both trackers. Any ideas?
